README.md

```diff
@@ -88,16 +88,13 @@ This project requires at least the following to build:
 
 * C++17
 * CMake 3.25
+* (Test Only) GoogleTest
 
-This project pulls [Google Test](https://github.com/google/googletest)
-from GitHub as a development dependency for its testing framework,
-thus requiring an active internet connection to configure.
-You can disable this behavior by setting cmake option
+You can disable building tests by setting cmake option
 [`BEMAN_EXEMPLAR_BUILD_TESTS`](#beman_exemplar_build_tests) to `OFF`
 when configuring the project.
 
-However,
-some examples and tests will not be compiled
```

Comment:
![Screenshot from 2025-03-02 02-22-53](https://github.com/user-attachments/assets/ab3e12df-8c46-4c08-9f0f-b4fe27abceb2)

Here's the overdue reminder... I think you've done what you need to be reminded about.

Reply:
Thanks for making sure we don't forget this detail!